### DISCUSSION TITLE
externalize library with externals and externalsType as common js,, umd, , amd and module and var

### DISCUSSION THREAD

**Discussion by Shashank-Bhatt1** [webpack/webpack#18325] [Q&A]
Hi there,  I needed answer for this question. Could anyone please help on that?

https://stackoverflow.com/questions/78333969/webpack-external-library-with-externalstype-as-common-js-umd-amd-and-module

Thanks in advanced.

**Comment by alexander-akait** [comment]
Can you create minimum reproducible test repo?
